src/v2/view-builder/views/email/BaseAuthenticatorEmailView.js

```diff
@@ -91,9 +91,7 @@ const Body = BaseFormWithPolling.extend(Object.assign(
             model.trigger('clearFormError');
           }, 0);
         }
-        setTimeout(() => {
-          this.startPolling(this.options.appState.get('dynamicRefreshInterval'));
-        }, 60000);
+        this.startPolling(60000);
```

Comment:
that would be polling every 1 min ?

Reply:
It would for 1 min and revert back to interval based on the next polling response.